src/buffer/out/Row.hpp

```diff
@@ -36,6 +36,12 @@ class ROW final
 
     size_t size() const noexcept;
 
+    void SetWrapForced(const bool wrap) noexcept;

```

Comment:
Could just move the getters/setters up here per the other PR and chat recently about inline potential.

Reply:
no reason not to